src/style.css

```diff
@@ -1 +1,97 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+}
+
+.input {
+  font-family: Avenir, sans-serif;
```

Comment:
Consider adding fallback fonts to ensure better compatibility across different systems. For example, you can use `font-family: Avenir, Helvetica, Arial, sans-serif;`.